idiss/src/lib.rs

```diff
@@ -14,11 +15,13 @@ use concordium_base::{
     },
     ps_sig,
 };
-use pairing::bls12_381::{Bls12, G1};
 use serde::{Deserialize as SerdeDeserialize, Serialize as SerdeSerialize};
 #[cfg(feature = "nodejs")]
 use serde_json::ser::to_string;
 
+type Bls12 = ark_ec::bls12::Bls12<ark_bls12_381::Config>;
+type G1 = ArkGroup<G1Projective>;
```

Comment:
Fixed. I use `id::constants::ArCurve` and `id::constants::IpPairing` everywhere where `concordium_base` is imported. I also defined `id::constants::ArCurveG2` to avoid `arkworks` dependency where `G2` is used